pkgs/tools/package-management/nix/default.nix

```diff
@@ -64,17 +65,29 @@ common =
       preConfigure =
         # Copy libboost_context so we don't get all of Boost in our closure.
         # https://github.com/NixOS/nixpkgs/issues/45462
-        if is20 then ''
+	lib.optionalString is20 ''
```

Comment:
this still doesn't look properly indented.

Reply:
I finally got neovim to not indent with tabs just this morning. Sorry. Ran `grep -Pn '\t' on the file in question to make sure no more slipped through